hack/lib/golang.sh

```diff
@@ -558,6 +560,17 @@ kube::golang::build_binaries() {
     local use_go_build
     local -a targets=()
     local arg
+    
+    # Add any files with those //generate annotations in the array below.
+    readonly BINDATAS=( "${KUBE_ROOT}/test/e2e/framework/gobindata_util.go" )
+    for bindata in ${BINDATAS[@]}; do
+	    # Only try to generate bindata if the file exists, since in some cases
```

Comment:
I'm still a little confused by this - when would you be trying to do a build but not have `test/e2e/framework/gobindata_util.go`?